certs/minio/README.md

```diff
@@ -0,0 +1,16 @@
+# MinIO Certificates
+
+This certificate is generated using [certgen](https://github.com/minio/certgen) utility from MinIO.
+
+It was generated using following command:
+
+```bash
+certgen -host "127.0.0.1,localhost"
```

Comment:
If I use a different LAN IP to run everything from (such as 192.168.0.2), is this going to cause certification problems?

Reply:
So can you please check by running this branch and see if minio is running fine in docker? And you can access it: https://etherealengine.github.io/etherealengine-docs/docs/installation/install_troubleshooting/#accessing-minio-s3-storage-provider-running-in-local-docker

If it doesnot then you will need to generate certificate for your ip.